temba/channels/models.py

```diff
@@ -1310,30 +1313,43 @@ def send_jasmin_message(cls, channel, msg, text):
 
     @classmethod
     def send_junebug_message(cls, channel, msg, text):
-        from temba.msgs.models import WIRED
+        from temba.msgs.models import WIRED, Msg
 
         # the event url Junebug will relay events to
         event_url = 'https://%s%s' % (
             settings.HOSTNAME,
             reverse('handlers.junebug_handler',
                     args=['event', channel.uuid]))
 
+        is_ussd = channel.channel_type in Channel.USSD_CHANNELS
```

Comment:
No reason to iterate the list on every msg send if we can just test against the channel type.